js/homework.js

```diff
@@ -0,0 +1,157 @@
+var blueBtn = document.querySelector("#blueBtn");
+blueBtn.addEventListener("click", jumboBlue);
+
+function jumboBlue() {
+    var blue = document.querySelector(".jumbotron");
+    var blue1 = document.querySelector("#bikeButton.btn-primary");
+    var blue2 = document.querySelector("#volunteerButton.btn-secondary");
+
+    if (!blue.className.includes("blue")) {
+        blue.className += " blue";
+    }
+    if (!blue1.className.includes("bikeButtonOrange")) {
+        blue1.className += " bikeButtonOrange";
+    }
+    if (!blue2.className.includes("volunteerButtonBlack")) {
+        blue2.className += " volunteerButtonBlack";
+    }
+}
+
+var orangeBtn = document.querySelector("#orangeBtn");
+orangeBtn.addEventListener("click", jumboOrange);
+
+function jumboOrange() {
+    var orange = document.querySelector(".jumbotron");
```

Comment:
Might be better to create a var at the top of the class called jumbotron. Like that if you change hte classname you only need to change it once. 

Same applies for the bikeButton and volunteerButton buttons.